### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by myabc**
**What problem is this PR intended to solve?**

Fix various typos - some found via `codespell` and `typos` CLI (h/t @kianmeng with PR #3166)

**Have you included adequate test coverage?**

This PR does not change test coverage - existing tests should suffice.

**Does this change affect the behavior of either the C or the Java implementations?**

This PR does not introduce any changes in behavior.

### CODE REVIEW

File: ext/java/nokogiri/internals/c14n/UtfHelper.java [hunk 1]
```diff
@@ -22,11 +22,10 @@
 import java.io.OutputStream;
 import java.util.Map;
 
-public final class UtfHelpper
```
Comment (flavorjones) [inline ext/java/nokogiri/internals/c14n/UtfHelper.java:25]: You'll also need to update the `spec.files` in `nokogiri.gemspec` if this file name has changed.

File: ext/java/nokogiri/internals/c14n/UtfHelper.java [hunk 2]
```diff
@@ -22,11 +22,10 @@
...
+public final class UtfHelper
 {
 
-  private
-  UtfHelpper()
+  private UtfHelper()
```
Comment (flavorjones) [inline ext/java/nokogiri/internals/c14n/UtfHelper.java:28]: The `rake format` task will split the return type out onto a separate line again. Please preserve that newline.

**Review by flavorjones — CHANGES_REQUESTED** [review]
Thank you for submitting this! I've made a few comments.

File: ext/java/nokogiri/internals/c14n/UtfHelper.java (hunk 2 above)
Comment (myabc) [reply ext/java/nokogiri/internals/c14n/UtfHelper.java]: I've fixed the formatting. It looks like there are other C/Java files that don't conform to the style - at least when running Artistic Style 3.4.15.

**Comment by myabc** [comment]
@flavorjones many thanks for the feedback! I've updated the PR accordingly.

**Review by flavorjones — APPROVED** [review]

**Comment by flavorjones** [comment]
Thank you again!

**Pull request merged**